package.json

```diff
@@ -29,20 +29,24 @@
     "@angular/platform-browser": "^5.0.0",
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/router": "^5.0.0",
-    "@deck.gl/core": "^7.1.6",
-    "@deck.gl/google-maps": "^7.1.6",
-    "@deck.gl/layers": "^7.1.6",
+    "@deck.gl/core": "^7.3.15",
+    "@deck.gl/google-maps": "^7.3.15",
+    "@deck.gl/layers": "^7.3.15",
     "@turf/bbox": "^6.0.1",
+    "@types/crypto-js": "^3.1.44",
     "@types/d3-scale": "^1.0.11",
     "@types/gapi": "0.0.35",
-    "@types/googlemaps": "^3.29.2",
+    "@types/googlemaps": "^3.38.0",
     "angular-webstorage-service": "^1.0.2",
-    "codemirror": "^5.34.0",
+    "codemirror": "^5.52.2",
     "colorbrewer": "^1.1.0",
-    "core-js": "^2.4.1",
-    "d3-color": "^1.0.3",
+    "core-js": "^2.6.11",
+    "crypto-js": "^4.0.0",
```

Comment:
Running locally, I'm getting this warning:

```
Module not found: Error: Can't resolve 'crypto' in '/Users/donmccurdy/Projects/bigquery-geo-viz/node_modules/crypto-js'
```

Unsure what to make of that. What version of nodejs are you using, and is the latest copy of `package-lock.json` included in this PR? The app still runs, though, it's just a warning.

Reply:
Yes, I saw this warning as well. I tried to resolve it.  Based on this thread

https://stackoverflow.com/questions/54162297/module-not-found-error-cant-resolve-crypto
and 
https://github.com/tensorflow/tfjs/issues/494

I should be able to fix this by adding

  "browser": {
    "crypto": false
  }

to the package.json but the solution did not work for me. The warning seems harmless but it's annoying.

Reply:
Ok, yeah should be fine for now then!